privacyidea/lib/utils/export.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+log = logging.getLogger(__name__)
+
+EXPORT_FUNCTIONS = {}
+IMPORT_FUNCTIONS = {}
+
+
+def register_export_name(name):
+    def wrapped(func):
+        # TODO: this doesn't work since the logging configuration is not
+        #       initialized during imports
+        # log.debug('Registering export function {0!s} with name {1!s}'.format(func, name))
+        EXPORT_FUNCTIONS[name] = func
+        return func
+    return wrapped
+
+
+def register_import_name(name):
+    def wrapped(func):
+        # TODO: this doesn't work since the logging configuration is not
+        #       initialized during imports
+        # log.debug('Registering export function {0!s} with name {1!s}'.format(func, name))
+        IMPORT_FUNCTIONS[name] = func
+        return func
+    return wrapped
+
+
+def register_export(func):
```

Comment:
It is not clear to me, why you also have `register_export` and `register_import` which are not used anywhere - currently.

Reply:
This was from a previous test where i used the module name for the registration.
I'll make `register_export` and `register_import` the default with both functionality (passing a name or using the functions module name).

Reply:
Done in 426389b